### PR TITLE
Fix os.path.joins in EnergyPlus workflow for EP-Launch

### PR DESCRIPTION
**Pull request by JasonGlazer**
Address https://github.com/NREL/EP-Launch/issues/98 

### Review Checklist
This will not be exhaustively relevant to every PR.
 - [ ] Functional code review (it has to work!)
 - [ ] If defect, results of running current develop vs this branch should exhibit the fix
 - [ ] CI status: all green or justified
 - [ ] Performance: CI Linux results include performance check
 - [ ] Unit Test(s)
 - C++ checks:
   - [ ] Argument types
   - [ ] If any virtual classes, ensure virtual destructor included, other things
 - IDD changes:
   - [ ] Verify naming conventions and styles, memos and notes and defaults
   - [ ] Open windows IDF Editor with modified IDD to check for errors
   - [ ] If transition, add to input rules file for interfaces
   - [ ] If transition, add transition source
   - [ ] If transition, update idfs
 - [ ] If new idf included, locally check the err file and other outputs
 - [ ] Required documentation updates?
 - [ ] ExpandObjects changes?
 - [ ] If output changes, including tabular output structure, add to output rules file for interfaces

### CODE REVIEW

**Review by Myoldmopar — APPROVED** [review]

**Comment by Myoldmopar** [comment]
CI is already close to finishing this up so I'll let @nrel-bot-2 finish, then I'll merge this.

**Comment by JasonGlazer** [comment]
Waiting for the CI for workflow fixes :-(

**Pull request merged**
